tests/app/views/test_update.py

```diff
@@ -116,7 +116,7 @@ def test_should_delete_service_by_id(self, service):
 
     def test_should_return_404_if_no_service(self):
         response = self.client.delete(
-            '/test-index/delete/100')
+            '/test-index/services/not-an-id-that-exists')
```

Comment:
Can we add a test for `self.client.delete('/test-index/baddoctype/<valid_id>')` as well?